src/Components/Detailspage.js

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { getCompanyDetails } from '../redux/CompaniesList';
+import '../../styles/Detailspage.css';
+
+const Detailspage = () => {
+  const { companyDetails } = useSelector((state) => state.companies);
+  const dispatch = useDispatch();
+  const { smbl } = useParams();
+  useEffect(() => {
+    dispatch(getCompanyDetails(smbl));
+  }, [dispatch, smbl]);
+
+  return (
+    <div className="parent" data-testid="detailspage">
+      <div className="items">
+        <img className="image" src={companyDetails.image} alt="company" />
+        <p>
+          (
+          {companyDetails.symbol}
+          )
+        </p>
+        <h1>{companyDetails.companyName}</h1>
+        <table className="table">
+          <tbody>
+            <tr>
+              <th scope="col">CEO</th>
+              <td>{companyDetails.ceo}</td>
+            </tr>
+            <tr>
+              <th scope="col">Sector</th>
+              <td>{companyDetails.sector}</td>
+            </tr>
+            <tr>
+              <th scope="col">Price</th>
+              <td>{companyDetails.price}</td>
+            </tr>
+            <tr>
+              <th scope="col">Currency</th>
+              <td>{companyDetails.currency}</td>
+            </tr>
+            <tr>
+              <th scope="col">Full Time Employees</th>
+              <td>{companyDetails.fullTimeEmployees}</td>
+            </tr>
+            <tr>
+              <th scope="col">Country</th>
+              <td>{companyDetails.country}</td>
+            </tr>
+            <tr>
+              <th scope="col">City</th>
+              <td>{companyDetails.city}</td>
+            </tr>
+            <tr>
+              <th scope="col">Phone</th>
+              <td>{companyDetails.phone}</td>
+            </tr>
+            <tr>
+              <th scope="col">Address</th>
+              <td>{companyDetails.address}</td>
+            </tr>
+          </tbody>
+
+        </table>
+      </div>
+    </div>
+  );
+};
```

Comment:
- [ ] As required for the project, kindly make sure you follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))  for your DetailPage, as it should be close the to one from the template, including:
   * Typography: font face, size, and weight
   * Layout: `composition`, space between and around elements, text-alignment(`space between`), color,...

| Template design | Your Design |
| ---------------------- | ----------------- | 
| ![Screenshot from 2023-01-12 23-41-24](https://github.com/Ashnaali3255/Stock-Metrics-WebApp/assets/83042742/1db6f4a3-7e8d-4131-8c7d-e50ebfc6ec7d) | ![Screenshot from 2023-06-06 21-51-42](https://github.com/Ashnaali3255/Stock-Metrics-WebApp/assets/83042742/a57188c3-eea7-4b4e-843d-75b3415f7f50) |

Reply:
Finally, I did it.